### ISSUE TITLE
Multitasking window

### ISSUE THREAD

**Issue by Eyion** [ferdium/ferdium-app#1882]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/ferdium/ferdium-app/blob/HEAD/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22enhancement+%3Asparkles%3A%22) for a feature request that matches the one I want to file, without success.

### Problem Description

Applications under the same account cannot open multiple windows like browsers.

### Proposed Solution

The same account application opens multiple windows like a browser. For example, if you log in to GitHub with account A, you want to be able to open multiple project pages in account A at the same time.

### Alternatives Considered

None

### Additional Information

![image](https://github.com/user-attachments/assets/d850e8a5-489a-4b15-b662-11bb0df2f128)

**Comment by SpecialAro** [comment]
Hi @Eyion 👋 thank you for logging this feature request

Just to try and understand it better, what you are suggesting is something about opening URLs in a new window insider Ferdium and not in the regular browser?

Or

Is it that two or more services should be able to share the same session data: for instance, YouTube service and Gmail service should use the same account without the need of login twice?

**Comment by Eyion** [comment]
@SpecialAro Yes, it should be closer to the first statement. If I want to open multiple GitHub projects in Ferrium simultaneously, how do I implement them? 
For example, I want to open a service and open multiple windows based on this service, just like I only need to open a browser, but I can open multiple tabs at the same time.

**Comment by theAkito** [comment]
A much needed feature. If this would be implemented, Ferdium would be pretty top-notch (well, after all the workspace problems are gone, as well).

Is there an estimate of how much effort & time this would take? How complicated is it, to implement this feature?
